src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,48 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchParams } from '../types/SearchParamsEnum';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get(SearchParams.Query) || '';
+  const sex = searchParams.get(SearchParams.Sex) || '';
+  const centuries = searchParams.getAll(SearchParams.Centuries) || [];
+
+  const allCenturies = ['16', '17', '18', '19', '20'];
+
+  const handleQuerySearch = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const search = getSearchWith(searchParams, {
+      query: event.target.value || null,
+    });
+
+    setSearchParams(search);
+  };
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        <SearchLink
+          params={{ sex: null }}
+          className={classNames({ 'is-active': !sex })}
+        >
+          All
+        </SearchLink>
+        <SearchLink
+          params={{ sex: 'm' }}
+          className={classNames({ 'is-active': sex === 'm' })}
```

Comment:
Consider will be better if you create a variable or enum for these magic words and use them everywhere